.travis.yml

```diff
@@ -19,21 +16,29 @@ addons:
       - artifacts/$TRAVIS_COMMIT
     s3_region: us-west-1
     permissions: public-read
+script: make test-ci
 env:
   global:
     - secure: U0dvpEjblZ3i+knw/cOWWgLvHUz/L9vN7Kv5VKDJ5ydvLy/1Tcueokp4hL7jhu4zwzlJto6MdFG1uKXr97zs+ETV9clYtRFBF/4YlAGs8sQcvPu4BOyH9pUJ77tnNR2ZX62wik9UFl6LmJOBpCjVkp+kh6it44QJm4Zv03Yx47w=
     - secure: wrAUnHRseJQrRUqoKWCY/Sw8Wd21kssvYiQ26HpjNc8Q3+uQY6IAMwyyyRRbr65nwHnYrk6KJfSEHYgmQbMeie/3+LseXbDGd9BmWAvujI2ZXNPjFxhSA4gmBvVtT/57fZaCdc8pAnGLb8OwiBSgQN0UJ55Kr59Yjhi3ehBZd1Q=
     - secure: yIrHdkLhWtO37N24cRU71PK80zvkPzlJ1Jm+1MS3C6/icgJ8At1N/UgpBdxuBs1tlJm1O7Ahk+/MhpGWlspx9tCF7Hzw9Rj0xpKhqsvM155zgH582xNPwNebiCr/tmsH2/ELgFBBgC9sCW5luzeKx9nyuedGA4fmD2fK5Phc5EE=
     - secure: fGRo+RTDcnWNnYjSgDtHIRFrkXDxYzFrF0JaGs73ZunLXoYbJ5FclaM9+wjw8BbGEEsVbyBYMSHyy8E2Y0VRggal/XlFkKaqQQHSDVuslcM5YdAkcLC8jFSuSzP32X6cY+L9fKszFVaO+vn84lXwiQnqgpOHAtN32/eB//5Ky1s=
     - secure: m9ylFoxkeoOjBTYi7IF93Rqq95uJkL95onYJc5k1Xo2wNCDrAfvZ8cv6o70Qr/lpWqvgxPueDP6VXjxKAmSueQnx9uGKyigFCpMH4zl7+7n7l4382Eeyex2uE6kCh/lA4EjkmE3QdIW6l8YCGJD1IoHqVSKPZ67ZSdnapTgGPYQ=
-  matrix:
-    - BROWSER=chrome_headless ARTIFACTS_BUCKET=recurly-js REPORT_COVERAGE=true
-    - BROWSER=chrome
-    - BROWSER=firefox
-    - BROWSER=safari
-    - BROWSER=edge
-    - BROWSER=ie_11
-    - BROWSER=ios_12
-    - BROWSER=android_9
-    - BROWSER=android_8
-    - BROWSER=android_7
+jobs:
+  include:
+    - stage: Lint
+      script: make lint
+    - stage: Test
+      env: BROWSER=chrome_headless REPORT_COVERAGE=true
+    - env: BROWSER=chrome
+    - env: BROWSER=firefox
+    - env: BROWSER=safari
+    - env: BROWSER=edge
+    - env: BROWSER=ie_11
+    - env: BROWSER=ios_12
+    - env: BROWSER=android_9
+    - env: BROWSER=android_8
+    - env: BROWSER=android_7
+    - stage: Publish
+      env: ARTIFACTS_BUCKET=recurly-js
+      script: make build
```

Comment:
I moved the artifact publication from being dependent upon passage of the chrome headless run to being dependent on the passage of all runs. In effect, nothing will change, as deploys were never possible without full suite passage. But this heightens the automated circuit breaker.